### PR TITLE
Switch to MSBuild

### LINKED ISSUE #36: Switch to msbuild on Windows
According to https://stackoverflow.com/a/5005814 , with msbuild you don't need a full Visual Studio installed.

Here's a outdated branch https://github.com/messense/mupdf-rs/tree/msbuild that tries to do it, needs a rebase and figure out how to pass build arguments.

### PR DESCRIPTION
**Pull request by Bobo1239**
Seems to work on my machine (without Visual Studio installed) but more testing welcome.

Closes #36.

### CODE REVIEW

File: mupdf-sys/build.rs [hunk 1]
```diff
@@ -305,21 +293,20 @@ fn build_libmupdf() {
         if cfg!(not(feature = "js")) {
             cl_env.push("/DFZ_ENABLE_JS#0".to_string());
         }
-        let d = cc::windows_registry::find(target.as_str(), "devenv.com")
+        let d = cc::windows_registry::find(target.as_str(), "msbuild.exe")
             .unwrap()
```
Comment (messense) [inline mupdf-sys/build.rs:297]:
```suggestion
        msbuild
```

File: mupdf-sys/build.rs [hunk 2]
```diff
@@ -305,21 +293,20 @@ fn build_libmupdf() {
...
-                "/build",
-                &format!("{}|{}", profile, msvc_platform),
-                "/project",
-                "libmupdf",
+                "/target:libmupdf",
+                &format!("/p:Configuration={}", profile),
+                &format!("/p:Platform={}", msvc_platform),
             ])
             .current_dir(&build_dir)
             .env("CL", cl_env.join(" "))
```
Comment (messense) [inline mupdf-sys/build.rs:305]: Does MSBuild uses this env var? I'm not sure.

Comment (Bobo1239) [reply mupdf-sys/build.rs:304]: The env var should directly affect the compiler (CL.exe) so should be independent of how it gets invoked. Also a quick test seems to indicate that disabling support for some formats does indeed work.

Comment (messense) [reply mupdf-sys/build.rs:304]: Thanks for confirming!

**Review by messense — APPROVED** [review]

**Pull request merged**